pip/__init__.py

```diff
@@ -155,6 +156,39 @@ def create_main_parser():
     return parser
 
 
+def _get_autocorrect_configuration_values(config):
+    di = dict(config.items())
+    configuration_problems = []
+
+    def get_float(var_name, default):
+        if var_name not in di:
+            return default
+
+        val = di[var_name]
+        try:
+            return float(val)
+        except Exception:
+            configuration_problems.append(
+                var_name + " should be a floating point value"
+            )
+            return None
+
+    wait_time = get_float("wait_time", 2.0)
```

Comment:
That is why I have included the fallback values (2.0 for wait_time here).

Is there a better way to have fallbacks for these?